### ISSUE TITLE
Disallow creating new directory, only choose existing one.

### ISSUE THREAD

**Issue by TheOddler** [passy/Android-DirectoryChooser#70]
I need the user to just select an existing folder, not create a new one. There doesn't seem to be an option for this, but please correct me if I'm wrong.

Thanks!

**Comment by passy** [comment]
Yes, that's right. There's no option for that at the moment as that was not something I needed. I'd be open to a pull request, though! :)

**Comment by TheOddler** [comment]
Ah, I'm very new to android development (only been at it a few hours :P), so I'm not sure I'm a good person to try this. But I'll see what I can do :D

**Comment by passy** [comment]
Cool! There seem to be a bunch of other libraries that do roughly the same as this one. Maybe one of them is closer to what you need. :)

**Comment by TheOddler** [comment]
Aha, I'll google a bit more then too :P Thanks!
